src/themes/website/styles/forms.scss

```diff
@@ -130,16 +130,16 @@
     }
 }
 
-input[type=search] {
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    @extend .icon-svg-magnifier;
-    background-position: 5px center;
-    background-size: 1em;
-    background-repeat: no-repeat;
-    padding-left: calc(1em + 10px);
-}
+// input[type=search] {
+//     white-space: nowrap;
+//     overflow: hidden;
+//     text-overflow: ellipsis;
+//     @extend .icon-svg-magnifier;
+//     background-position: 5px center;
+//     background-size: 1em;
+//     background-repeat: no-repeat;
+//     padding-left: calc(1em + 10px);
+// }
```

Comment:
shouldn't this be rather deleted then commented out?

Reply:
I would like to leave it like this for a little while to double check that we don't break anything. Then delete this of course.